docs/guides/model-your-first-process.md

```diff
@@ -16,24 +16,30 @@ import SaasPrereqs from './react-components/\_saas-prerequisites.md'
 
 ## Design and deploy a process
 
-In Camunda 8, you have two options to design and deploy a process, but for the purposes for this guide you will find instructions for [Web Modeler](../components/modeler/about-modeler.md).
+With Camunda 8, you can design and deploy a process using both [Desktop Modeler and Web Modeler](/components/modeler/about-modeler.md). For this guide, we will use [Web Modeler](../components/modeler/about-modeler.md).
 
-1. From Modeler, click **New project**.
+1. From Web Modeler, select **Create new project**.
 2. Name your project and select **Create new > BPMN diagram**.
    ![blank project create bpmn diagram](./img/blank-project.png)
-3. Give your model a descriptive name, and then give your model a descriptive ID within the **General** tab inside the properties panel on the right side of the screen.
-4. Create a task by dragging the rectangular task icon from the palette, or by clicking the existing start event and clicking the task icon. Make sure there is an arrow connecting the start event to the task.
+3. Give your model a descriptive name and ID within the **General** tab inside the properties panel on the right side of the screen. For example, you may name the diagram `Process invoice` with an ID of `process_invoice`.
+4. Create a task by dragging the rectangular task icon from the palette, or by clicking the existing start event, hovering over the `+` icon, and selecting the task icon. Ensure an arrow connects the start event to the task.
+
+:::note
+So far, we have included two BPMN elements in our diagram: a start event and a task. [Events](/components/modeler/bpmn/events.md) in BPMN represent things that happen. A process can react to events and emit events, for example. The basic elements of BPMN processes are [tasks](/components/modeler/bpmn/tasks.md), or atomic units of work composed to create a meaningful result. Whenever a token reaches a task, the token stops and Zeebe creates a job and notifies a registered worker to perform work. Review the [complete list of supported BPMN elements](/components/modeler/bpmn/bpmn-coverage.md)
+:::
+
 5. Name the task by double-clicking the task or using the properties panel.
 6. Create an end event by dragging the end event icon from the palette, or by clicking the existing start event and clicking the end event icon.
-7. No need to save. Web Modeler will autosave every change you make.
+
```

Comment:
We never actually talk through deploying and running a process, and what this means. Need to include these details.